src/pudl/metadata/fields.py

```diff
@@ -1728,15 +1729,15 @@
     },
     "operator_name": {
         "type": "string",
-        "descripion": "The name of the EIA operator utility.",
+        "description": "The name of the EIA operator utility.",
     },
     "operator_state": {
         "type": "string",
         "description": "The state where the operator utility is located.",
     },
     "operator_utility_id_eia": {
         "type": "integer",
-        "descrption": "The EIA utility Identification number for the operator utility.",
+        "description": "The EIA utility Identification number for the operator utility.",
```

Comment:
Typos which were not causing errors because these fields aren't actually used anywhere, and so are never instantiated.